web/docs/testing.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Testing
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import AddExternalAuthEnvVarsReminder from './_addExternalAuthEnvVarsReminder.md'
+
+:::info
+Wasp is in beta, so keep in mind there might be some kinks / bugs, and possibly some changes with testing support in the future.
+If you encounter any issues, reach out to us on [Discord](https://discord.gg/rzdnErX) and we will make sure to help you out!
+:::
+
+# Web App
+
+Wasp enables you to quickly and easily write both unit tests and React component tests for your frontend code. Becase we already use [Vite](https://vitejs.dev/), we were able to add many additional testing capabilities by giving you an integrated [Vitest](https://vitest.dev/) experience out of the box.
```

Comment:
```suggestion
Wasp enables you to quickly and easily write both unit tests and React component tests for your frontend code. Because we already use [Vite](https://vitejs.dev/), we also support testing the web app by giving you an integrated [Vitest](https://vitest.dev/) experience out of the box.
```